test/emulation.spec.js

```diff
@@ -308,7 +308,7 @@ module.exports.describe = function({testRunner, expect, playwright, headless, FF
       expect(await page.evaluate(() => navigator.language)).toBe('fr-CH');
       await context.close();
     });
-    it('should format number', async({browser, server}) => {
+    it.fail(FFOX)('should format number', async({browser, server}) => {
```

Comment:
why doesn't this work?

Reply:
We need to patch builtins w/ locale data.